core/admin.py

```diff
@@ -66,6 +89,9 @@ class SubmissionAdmin(admin.ModelAdmin):
 
     actions = [rescore_submission_with_notification, rescore_submission_without_notifying]
 
+    def task(self, obj: Submission):
+        return obj.approach.task
```

Comment:
Will this be expensive? Can we prefetch related?

Reply:
It somehow already is, I'm guessing because approach is in the fields list.